<launch>
    <arg name="namespace" default="yk_creator"/>
    <arg name="start_trial" default="0"/>
    <group ns="$(arg namespace)">
        <node pkg="vibro_tactile_toolbox" name="collect_tactile_data" type="collect_tactile_data.py" output="screen">
            <param name="namespace" value="$(arg namespace)"/>
            <param name="root_pwd" value="$(find vibro_tactile_toolbox)"/>
            <param name="start_trial" value="$(arg start_trial)"/>
        </node>
    </group>
</launch>